client/rest-high-level/src/main/java/org/elasticsearch/client/rollup/GetRollupJobResponse.java

```diff
@@ -51,6 +51,15 @@
     static final ParseField STATE = new ParseField("job_state");
     static final ParseField CURRENT_POSITION = new ParseField("current_position");
     static final ParseField UPGRADED_DOC_ID = new ParseField("upgraded_doc_id");
+    static final ParseField BULK_LATENCY = new ParseField("bulk_latency_in_ms");
+    static final ParseField SEARCH_LATENCY = new ParseField("search_latency_in_ms");
+    static final ParseField SEARCH_FAILURES = new ParseField("search_failures");
+    static final ParseField BULK_FAILURES = new ParseField("bulk_failures");
+    static final ParseField MIN = new ParseField("min");
+    static final ParseField MAX = new ParseField("max");
+    static final ParseField AVG = new ParseField("avg");
+    static final ParseField COUNT = new ParseField("count");
+    static final ParseField TOTAL = new ParseField("total");
```

Comment:
To be consistent with the `_stats` API can we call these `bulk_time_in_millis` and `query_time_in_millis` ? I am also not sure if we need the `min`, the `max` and the `avg`. It should be enough to have the total time spent in these operations and the number of calls per action ?

Reply:
@polyfractal Are `MIN`, ..., ..., `TOTAL` are leftovers from previous iterations? They look unused to me.

Reply:
Right you are!